src/inputs/selectNext/selectNext.tsx

```diff
@@ -0,0 +1,439 @@
+                    fontWeight: theme.typography.fontWeightMedium,
+                    margin: `${selectOptionMarginVertical}px 0`,
+                    padding: `${selectOptionPaddingVertical}px ${selectOptionPaddingHorizontal}px`,
+                    whiteSpace: 'nowrap',
+                    '&--is-selected, &--is-focused, &:hover': {
+                        backgroundColor: selectOptionSelectedBg,
+                        color: `${p.text.primary} !important`,
+                    },
+                },
+            },
+        },
+        root: {
+            display: 'inline-block',
+            minWidth: 200,
+            position: 'relative',
+        },
+        label: {
+            marginBottom: 8,
+        },
+    };
+});
+
+/**
+ * Define custom Arrow Component
+ */
+const CustomArrow = (componentProps) => (
+    <components.DropdownIndicator
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...componentProps}
+    >
+        <div>
+            <Icon
+                compact
+                size={16}
+                title="Select"
+                type="chevron-down"
+            />
+        </div>
+    </components.DropdownIndicator>
+);
+
+/**
+ * Define custom MenuList Component
+ */
+const CustomMenuList = (componentProps) => {
+    const {
+        children,
+        innerRef,
+        selectProps,
+    } = componentProps;
+
+    return (
+        <components.MenuList
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...componentProps}
+            ref={innerRef}
+        >
+            <ScrollBar
+                autoHeight
+                autoHeightMax={selectProps.dropdownMenuMaxHeight || 180}
+                autoHeightMin={selectProps.dropdownMenuMinHeight}
+                autoHide
+                className="select-menu-scrollbar"
+                ref={selectProps.menuScrollBarRef}
+            >
+                {children}
+            </ScrollBar>
+        </components.MenuList>
+    );
+};
+
+const CustomOption = (componentProps) => {
+    const {
+        children,
+        className,
+        innerRef,
+        isSelected,
+        selectOption,
+    } = componentProps;
+
+    const optionClass = ClassNames(
+        className,
+    );
+
+    return (
+        <components.Option
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...componentProps}
+            aria-selected={isSelected}
+            className={optionClass}
+            selectOption={selectOption}
+            ref={innerRef}
+            tabIndex={0}
+        >
+            {children}
+        </components.Option>
+    );
+};
+
+type Option = {
+    value: string;
+    label: string;
+};
+
+/**
+ * The Select component represents a control that provides a menu of options.
+ */
+// eslint-disable-next-line prefer-arrow-callback
+const SelectNext = React.forwardRef(function SelectNext(
+    props: PropTypes, ref: LegacyRef<HTMLDivElement>,
```

Comment:
Shouldn't these be on separate lines? =)

Reply:
Done!

Reply:
Gracias!  Resolving.